packages/ResponsesAccordion/src/components/Responses/Responses.styles.js

```diff
@@ -0,0 +1,35 @@
+import { css } from "styled-components";
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+
+const responsesStyles = css`
+  ${stylers.lineHeight()}
+
+  color: ${tokens.textColor.subtle};
+  margin: ${tokens.space} 0;
+  max-height: 420px;
```

Comment:
Do we want a larger max height since this is going inside a scrolling div and can also be collapsed? Double scrollbar in a sidepanel might not be the greatest. We won't be able to avoid it in all cases but maybe it should be larger to allow more content to be shown before the second scrollbar appears?

Reply:
That's a good question.... for @krismckinnonacl!
Luckily he already gave me his answer.  He was happy with this height.

Reply:
Nice trolling tagging the person that doesn't need to be tagged 😄

Reply:
Ha. He's told me he doesn't get the notifications anyway. And if he does, :trollface: